src/features/activity/AddressActivityItem.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { StyleSheet } from 'react-native'
+import { useTranslation } from 'react-i18next'
+import Box from '../../components/Box'
+import Text from '../../components/Text'
+import { useColors } from '../../theme/themeHooks'
+import TouchableOpacityBox, {
+  TouchableOpacityBoxProps,
+} from '../../components/TouchableOpacityBox'
+import AccountIcon from '../../components/AccountIcon'
+import { useAccountStorage } from '../../storage/AccountStorageProvider'
+import Menu from '../../assets/images/menu.svg'
+import { ellipsizeAddress } from '../../utils/accountUtils'
+
+export const LIST_ITEM_HEIGHT = 70
+export type AddressActivityItemProps = {
+  accountAddress: string
+  showBubbleArrow?: boolean
+  onMenuPress?: () => void
+} & TouchableOpacityBoxProps
+
+const AddressActivityItem = ({
+  accountAddress,
+  showBubbleArrow,
+  onMenuPress,
+  ...rest
+}: AddressActivityItemProps) => {
+  const colors = useColors()
+  const { currentAccount } = useAccountStorage()
+  const { t } = useTranslation()
+
+  return (
+    <Box>
```

Comment:
Can the background color and `{...rest}` just be applied to the root `<Box>`?

Reply:
Hmm yeah the reason I did this was because I didn't want those styles being applied to where the bubble arrow was but i'm sure I can work around it. Will look into it

Reply:
Done